src/components/dialogs/name-check-issues.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-dialog v-model="display" width="45rem" persistent :attach="attach">
+    <v-card>
+      <v-row no-gutters>
+        <v-col cols="6" class="text-end">
+          <v-icon class="mr-n4" :color="options.iconColor" size="2.5rem">{{ options.icon }}</v-icon>
+        </v-col>
+        <v-col>
+          <v-btn cols="6" class="close-btn outlined" icon :ripple="false" size="2.5rem" @click="proceed(false)">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+        </v-col>
+      </v-row>
+      <p class="dialog-title ma-0 pt-5">
+        {{ options.title }}
+      </p>
+      <v-card-text class="pt-10">
+        <p v-for="line, index in options.text"
+            :key="index"
```

Comment:
Might want to expand the id just a little to keep it unique, ie `options-text-${index}` or something like that? 
Just in the event if you had multiple keys using index, it could throw errors. 
Pretty minor!

Reply:
Oh interesting, I didn't understand how that worked. So it sets the id to the key binding? So you're saying I should change the key binding to that right?

Reply:
Absolutely, it doesn't have to be the exact same as the id, just a means of making it unique from other potential `:key=index` values.